list.java

```diff
@@ -0,0 +1,38 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+import org.json.simple.JSONArray; 
+import org.json.simple.JSONObject; 
+import org.json.simple.parser.*; 
+
+public class list {
+
+	public list(String listType) throws FileNotFoundException, IOException, ParseException {
+		//Create list of files in saves folder
+		File f = new File(System.getProperty("user.dir") + "\\save");
+		String[] contents = f.list();
+		String saveFile = null;
+
+		// Find umlSave.json file
+		for (int x = 0; x < contents.length; ++x) {
+			if (contents[x].equals("umlSave.json")){

```

Comment:
Does this mean that the saved UML file must be named "umlSave?" Is that a requirement we are adding?

Reply:
Yeah.  Both me and Connor didn't understand how to implement that.  How would we know what the saved file's title will be, especially since the user can make multiple saved files.  We just assumed that we'd have our own json file separate of the user that would hold all the information.  Do you know how to avoid this? Cause that would be much better if we didn't have our own separate save file.

Reply:
I'm also confused by this, is there a reason we're saving a file without the user specifying it?

Reply:
This part is outdated.  I originally tried to find the save file for the information, but changed it to just use the class.  You should be able to see the updated version at the bottom of this pull request.